content/technotes/2019-01-10-hugo.md

```diff
@@ -0,0 +1,101 @@
+
+* Community calls now being similar to blog posts and tech notes, they're cross-listed via their tags and authors so are more likely to be found.
+
+# Feed updates
+
+If you want to keep up with our production, you can subscribe to different feeds (or [our newsletter](https://news.ropensci.org/) since it includes mentions of recent posts!).
+
+## All the feeds
+
+Our website has both RSS and JSON feeds. We added a JSON feed because we could and because some people might like that. 
+
+* RSS feeds https://ropensci.org/blog/index.xml, https://ropensci.org/technotes/index.xml
+
+* JSON feeds https://ropensci.org/blog/index.json, https://ropensci.org/technotes/index.json
+
+Here's [how we added the JSON feeds](https://github.com/ropensci/roweb2/pull/315).
+
+## Not too much content for R-Bloggers
+
+R-Bloggers feed https://ropensci.org/rbloggers/index.xml
+
+We created an [R-Bloggers page which only outputs XML](https://github.com/ropensci/roweb2/blob/master/content/rbloggers/_index.md), with [an XML template](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/rbloggers/rss.xml) ensuring this RSS feed is a full RSS feed contrary to the rest of our website that uses Hugo default RSS feeds with excerpts. We do not filter posts by category which means this not-so-R post (but still a bit R, thanks to our explaining how to browse our R stuff ;o) will appear in R-Bloggers, but we might do that later on.
+
+# Twitter friendliness
+
+We're aware that our content does not only get shared on Twitter, but this social media platform is an important player so we've upped our Twitter game with two edits.
+
+{{< tweet 1073519094958313472>}}
+
+* We've worked on adding Twitter metadata to our [head partial](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/partials/head.html) so when tweeting links from the website you get a pretty card (see above). We've used [resources listed in this GitHub issue](https://github.com/ropensci/roweb2/issues/290#issue-367542204) and [Twitter card validator](https://cards-dev.twitter.com/validator) on Netlify preview links. Whilst updating the head partial we realized not all our pages used it so we homogeneized all pages. Some of our pages like [about](/about) have fancy stuff that's not Markdown based, but we now handle that with partials listed in YAML and then called from the [single.html default template](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/_default/single.html). Here [the source of our about page](https://github.com/ropensci/roweb2/blob/master/content/about.md).
```

Comment:
This paragraph has ended up with different linespacing somehow

Reply:
I've removed the itemize here, and opened an issue about this #380